.github/libs/versionUpdater.js

```diff
@@ -0,0 +1,98 @@
+const semanticVersionLevels = {
+    major: 'MAJOR',
+    minor: 'MINOR',
+    patch: 'PATCH',
+    build: 'BUILD',
```

Comment:
Please rename this constant to `UPPER_SNAKE_CASE`

```suggestion
    BUILD: 'BUILD',
```

Reply:
Hey @roryabraham, are you sure you want to change that?

I am happy to change it but these constants are inside an object, do you want me change this anyway?

Reply:
Changed